### DISCUSSION TITLE
Self hosted with "https" After entering login credentials during authorization getting 401 unauthorisation error.

### DISCUSSION THREAD

**Discussion by Mallikarjun-A** [hoppscotch/hoppscotch#3188] [Q&A] [answered]
We have Self hosted deployment with docker images on AWS. after performing following steps we are facing 401 unauthorization error.

Access the Front end UI >> click on login >> click on microsoft login >> login credentials page is displayed >> Enter Username and Password >> click login >> keep signed pop up  >> select option no  >> 401 unauthorized error on UI (at this web url is **https://sub-domain.com/v1/auth/microsoft/callback?code=0."some string"&state="some string"&session_state="some string""**)
Here "some string" referring to alphanumeric value for security purpose have not given.

Note : The settings configured in Microsoft Azure for login is working fine http and hosted with local host and same settings is used for https.

Can some one help with this what is the issue here ?

@AndrewBastin @abraham @silentmatt @kant @dunglas @

**Comment by joramk** [comment]
Got the same issue with GitHub login. Even E-Mail login is not working for me (frontend and admin backend). I am running Hoppscotch with Docker behind HAProxy and I am redirecting the /v1 requests to the backend instance port 3000.

The callback URL _https://hoppscotch.example.org/v1/auth/github/callback?code=1111111111111111111&state=2222222222222222_ shows this in the browser: `{"statusCode":401,"message":"Unauthorized"}`

No error is shown in the docker logs for any instance. I have no clue what the problem is.

**Reply by joramk** [reply, marked as answer]
Seems ok now. I have rebuild the docker images from scratch and cleared all cookies. Now it works so far.  Still got a problem with joining a team.
